release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2892,6 +2892,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.29
+[id="ocp-4-17-29_{context}"]
+=== RHSA-2025:4723 - {product-title} {product-version}.29 bug fix and security update advisory
+
+Issued: 15 May 2025
+
+{product-title} release {product-version}.29 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4723[RHSA-2025:4723] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4433[RHBA-2025:4725] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.29 --pullspecs
+----
+
+[id="ocp-4-17-29-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a race condition in the hostname that handled the code caused inconsistencies between the boot and machine hostnames. With this release, the race condition is resolved, which ensures consistent hostnames in the Ignition configuration file during the operating system installation. (link:https://issues.redhat.com/browse/OCPBUGS-55680[OCPBUGS-55680]). 
+
+* Previously, the Subject Alternative Name (SAN) of the custom certificate that the user added to the `hc.spec.configuration.apiServer.servingCerts.namedCertificates` field conflicted with the hostname set in the `hc.spec.services.servicePublishingStrategy` field for the KAS. As a result, the KAS certificate was not added to the set of certificates to generate a new payload, causing new nodes that joined the HostedCluster to have validation problems with the certificates. With this release, the validation will fail earlier so that the user is warned about the issue with the conflicting SANs. (link:https://issues.redhat.com/browse/OCPBUGS-55500[OCPBUGS-55500]). 
+
+* Previously, the boot image updates failed when the AMI for a specified region was not found because, in OKD, Amazon Machine Images (AMIs) for all regions are not published in the `scos.json` file for the installer. With this release, update failures do not occur because `us-east-1` is the default region that is used for the unavailable regions during Amazon Web Services (AWS) boot image updates. (link:https://issues.redhat.com/browse/OCPBUGS-55490[OCPBUGS-55490]).
+
+* Previously, when viewing the list of installed operators and the currently selected project matches an operator's default namespace while copied CSVs are disabled in OLM, an operator appeared twice in the list. With this release, the operator appears once. (link:https://issues.redhat.com/browse/OCPBUGS-55415[OCPBUGS-55415]).
```

Comment:
* Operator is always capitalized
* Can that first sentence be broken into 2-3? it's quite long and complex.
* Define OLM and CSV?
```suggestion
* Previously, when viewing the list of installed Operators and the currently selected project matches an Operator's default namespace while copied CSVs are disabled in OLM, an Operator appeared twice in the list. With this release, the Operator appears once. (link:https://issues.redhat.com/browse/OCPBUGS-55415[OCPBUGS-55415]).
```